app/src/main/res/drawable/minus_background.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
```

Comment:
Is it really required to use the layer-list ?
You can set the circular background to Image button using android:background attribute.
And the Image button icon can be set using android:src attribute.